tests/integration-tests/tests/schedulers/test_slurm_accounting.py

```diff
@@ -262,3 +273,38 @@ def _check_cluster_external_dbd(cluster, config_params, region, scheduler_comman
         headnode_remote_command_executor,
     )
     _test_jobs_get_recorded(scheduler_commands)
+
+
+def _check_inter_clusters_external_dbd(cluster_1, cluster_2, scheduler_commands_factory):
+    headnode_remote_command_executor_1 = RemoteCommandExecutor(cluster_1)
+    scheduler_commands_1 = scheduler_commands_factory(headnode_remote_command_executor_1)
+    headnode_remote_command_executor_2 = RemoteCommandExecutor(cluster_2)
+    scheduler_commands_2 = scheduler_commands_factory(headnode_remote_command_executor_2)
+
+    job_ids = []
```

Comment:
Why keep the index as 20?

Also can we add an explanation or Function definition